From 54e273594e716e6c1b0dbec1f9a1c856a876edea Mon Sep 17 00:00:00 2001
From: "Marco A. Gutierrez" <marco@openrobotics.org>
Date: Thu, 7 Jan 2021 17:35:06 +0800
Subject: [PATCH] removing tests

Signed-off-by: Marco A. Gutierrez <marco@openrobotics.org>
---
 setup.cfg | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index e54e17e..68a4e6c 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -4,9 +4,7 @@ script-dir=$base/lib/building_map_tools
 install-scripts=$base/lib/building_map_tools
 
 [options]
-setup_requires = pytest-runner
 tests_require = pytest
 
 [tool:pytest]
 addopts = --verbose
-testpaths = test
\ No newline at end of file
-- 
2.25.1

